tests/test_spawn.py

```diff
@@ -11,118 +11,203 @@
 
 # load app version of chart
 here = os.path.dirname(os.path.abspath(__file__))
-chart_yaml = os.path.join(here, os.pardir, 'jupyterhub', 'Chart.yaml')
+chart_yaml = os.path.join(here, os.pardir, "jupyterhub", "Chart.yaml")
 
 with open(chart_yaml) as f:
     chart = yaml.safe_load(f)
-    jupyterhub_version = chart['appVersion']
+    jupyterhub_version = chart["appVersion"]
+
 
 def test_api(api_request):
     print("asking for the hub's version")
-    r = api_request.get('')
+    r = api_request.get("")
     assert r.status_code == 200
     assert r.json().get("version", "version-missing") == jupyterhub_version
 
+    """kubectl logs deploy/hub - on a successful run
```

Comment:
Is this part of the test?

Reply:
I updated the doc strings to reflect what this is about. 

```python
def test_api(api_request):
    """
    Tests the hub api's root endpoint (/). The hub's version should be returned.
    A typical jupyterhub logging response to this test:
        
        [I 2019-09-25 12:03:12.051 JupyterHub log:174] 200 GET /hub/api (test@127.0.0.1) 9.57ms
    """
```